sql_generators/glean_usage/__init__.py

```diff
@@ -97,19 +97,21 @@ def generate(
     elif exclude:
         table_filter = partial(table_matches_patterns, exclude, True)
 
-    baseline_tables = list_baseline_tables(
-        project_id=target_project,
-        only_tables=[only] if only else None,
-        table_filter=table_filter,
-    )
-
-    # filter out skipped apps
-    baseline_tables = [
-        baseline_table
-        for baseline_table in baseline_tables
-        if baseline_table.split(".")[1]
-        not in [f"{skipped_app}_stable" for skipped_app in SKIP_APPS]
-    ]
+    def get_tables(table_name="baseline_v1"):
+        baseline_tables = list_tables(
+            project_id=target_project,
+            only_tables=[only] if only else None,
+            table_filter=table_filter,
+            table_name=table_name,
+        )
+
+        # filter out skipped apps
+        return [
+            baseline_table
+            for baseline_table in baseline_tables
+            if baseline_table.split(".")[1]
+            not in [f"{skipped_app}_stable" for skipped_app in SKIP_APPS]
+        ]
```

Comment:
There are some apps that do not have `baseline` tables. Event monitoring only depends on `event_v1` tables, so I added a `base_table_name` parameter to the table generators that gets passed into this function to get the corresponding stable tables.